rs/ethereum/cketh/test_utils/src/lib.rs

```diff
@@ -581,6 +580,70 @@ impl CkEthSetup {
         }
         self
     }
+
+    pub fn max_logs_block_range(&self) -> u64 {
+        if self.evm_rpc_id.is_none() {
+            799
+        } else {
+            500
+        }
+    }
+
+    pub fn received_eth_event_topic(&self) -> serde_json::Value {
+        if self.evm_rpc_id.is_none() {
+            serde_json::Value::String(RECEIVED_ETH_EVENT_TOPIC.to_string())
+        } else {
+            // The EVM-RPC canister models topics as `opt vec vec text`, see
+            // https://github.com/internet-computer-protocol/evm-rpc-canister/blob/3cce151d4c1338d83e6741afa354ccf11dff41e8/candid/evm_rpc.did#L69.
+            // This means that a simple topic such as `["0x257e057bb61920d8d0ed2cb7b720ac7f9c513cd1110bc9fa543079154f45f435"]`
+            // must actually be represented as `[["0x257e057bb61920d8d0ed2cb7b720ac7f9c513cd1110bc9fa543079154f45f435"]].
+            // The JSON-RPC providers seem to be able to handle both formats.
+            serde_json::Value::Array(vec![serde_json::Value::String(
+                RECEIVED_ETH_EVENT_TOPIC.to_string(),
+            )])
+        }
+    }
+
+    fn eth_get_logs_response_size_initial_estimate(&self) -> u64 {
+        100 + HEADER_SIZE_LIMIT
```

Comment:
Same question here: Where does the `100` come from?

Reply:
(This was answered in your corresponding comment in the production code)